server/blob/blob-cassandra/src/main/java/org/apache/james/blob/cassandra/cache/CassandraCacheConfiguration.java

```diff
@@ -77,6 +96,20 @@ public static Builder builder() {
         return new Builder();
     }
 
+    public static CassandraCacheConfiguration from(Configuration configuration) {
+        Optional<Duration> ttl = Optional.ofNullable(configuration.getString("cache.ttl", null))
```

Comment:
please use `cache.cassandra` prefix for properties that are not directly cache concepts (like timeout for example)